docs/es/actions.md

```diff
@@ -0,0 +1,178 @@
+
+En la práctica usaremos con frecuencia la [desestructuración de argumentos](https://github.com/lukehoban/es6features#destructuring) de ES2015 para simplificar nuestro código (especialmente cuando tengamos que llamar a `commit` múltiples veces).
+
+``` js
+actions: {
+  increment ({ commit }) {
+    commit('increment')
+  }
+}
+```
+
+### Ejecutar Acciones
+
+Las acciones pueden ser ejecutadas con el método `store.dispatch`:
+
+``` js
+store.dispatch('increment')
+```
+
+Esto puede parecer algo inútil a primera vista: si queremos incrementar el contador, ¿por qué no llamar a `store.commit('increment')` directamente? Recordemos que **las mutaciones deben ser síncronas**. Las acciones no se acogen a esta limitación por lo tanto pueden contener operaciones asíncronas:
+
+``` js
+actions: {
+  incrementAsync ({ commit }) {
+    setTimeout(() => {
+      commit('increment')
+    }, 1000)
+  }
+}
+```
+
+Las acciones pueden ser ejecutadas con el mismo formado de payload y estilo-objeto que las mutaciones:
+
+``` js
+// Ejecutar con payload
+store.dispatch('incrementAsync', {
+  amount: 10
+})
+
+// Ejecutar con un object
+store.dispatch({
+  type: 'incrementAsync',
+  amount: 10
+})
+```
+
+Un ejemplo más real puede ser una acción de confirmación de un carrito de compra, el cual puede incluir **llamas asíncronas a APIs** y **commitear múltiples mutaciones**.
+
+``` js
+actions: {
+  checkout ({ commit, state }, products) {
+    // Guardar referencia de los productos actualmente en el carrito
+    const savedCartItems = [...state.cart.added]
+    // Enviar una petición de confirmación, modo optimista
+    // Limpiar el carrito
+    commit(types.CHECKOUT_REQUEST)
+    // La API accepta callbacks de error y éxito
+    shop.buyProducts(
+      products,
+      // Gestión caso éxito
+      () => commit(types.CHECKOUT_SUCCESS),
+      // Gestión caso error
+      () => commit(types.CHECKOUT_FAILURE, savedCartItems)
+    )
+  }
+}
+```
+
+Hagase notar que estamos ejecutando un flujo de operaciones asíncronas y registrando los efectos laterales (modificaciones de estado) de la acción por medio de commits.
+
+### Ejecutar Acciones en Componentes
+
+Puedes ejecutar acciones en componentes con `this.$store.dispatch('xxx')` ó hacer uso del helper `mapActions` para mapear métodos del componente a llamadas tipo `store.dispatch` (esto require de la inyección de `store` en el Root).
+
+``` js
+import { mapActions } from 'vuex'
+
+export default {
+  // ...
+  methods: {
+    ...mapActions([
+      'increment', // Mapear this.increment() a this.$store.dispatch('increment')
+      
+      // mapActions también acepta payloads:
+      'incrementBy' // this.incrementBy(amount) se mapea a this.$store.dispatch('incrementBy', amount)
+    ]),
+    ...mapActions({
+      add: 'increment' // map this.add() to this.$store.dispatch('increment')
```

Comment:
Visto también `map this.add() to this.$store.dispatch('increment')` sin traducir.